x/feemarket/keeper/migrations.go

```diff
@@ -11,3 +17,15 @@ func NewMigrator(keeper Keeper) Migrator {
 		keeper: keeper,
 	}
 }
+
+func (m Migrator) Migrate1to2(ctx sdk.Context) error {
+	store := ctx.KVStore(m.keeper.storeKey)
+	baseFeeKeyPrefix := []byte{2}
+	bz := store.Get(baseFeeKeyPrefix)
+	if len(bz) > 0 {
```

Comment:
I'd also add a check if the paramset key is register

Reply:
what do you mean by registered, I think the paramset items are set directly?